test/nginx_opentracing_test.py

```
@@ -210,11 +262,24 @@ def testFastcgiPropagation(self):
         self.assertEqual(len(self.nginx_traces), 2)
 
     def testGrpcPropagation(self):
-        app = app_service.AppStub(self.grpcConn)
-        app.CheckTraceHeader(app_messages.Empty())
+        # Set up gRPC just for this test
+        grpc_conn = self._setup_grpc()
+        app = app_service.AppStub(grpc_conn)
+        try:
+            app.CheckTraceHeader(app_messages.Empty())
+        except Exception as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)